conans/client/tools/scm.py

```diff
@@ -129,6 +126,16 @@ def get_remote_url(self, remote_name=None):
             pass
         return None
 
+    def get_qualified_remote_url(self):
+        url = self.get_remote_url()
+        if os.path.exists(url):
```

Comment:
This code is executed only for the `Git` class, as it is overridden in SVN; so I won't do any logic here. I'll keep the `return self.get_remote_url()`.

Reply:
We need the logic only for Git, because SVN only give back an url (with file protocol for loacal repositories!)